randovania/gui/game_session_window.py

```diff
@@ -897,7 +892,7 @@ async def update_multiworld_client_status(self):
             if not self.multiworld_client.is_active:
                 you = self._game_session.players[self.network_client.current_user.id]
                 session_id = f"{self._game_session.id}_{self._game_session.game_details.seed_hash}_{you.row}"
-                persist_path = self.network_client.server_data_path.joinpath(f"game_session_{session_id}.json")
+                persist_path = self.network_client.server_data_path.joinpath(f"game_session_{session_id}")
```

Comment:
any particular reason for dropping the file extension?

Reply:
This path is now a directory. Though this is changing again later, likely